### PR TITLE
st1 [1994][ADD] stock_picking_type_kanban_view

### PR DESCRIPTION
**Pull request by kakurai8**
[1994](https://www.quartile.co/web#id=1994&model=project.task&view_type=form&menu_id=)

やりたいこと
- 「アクションの編集」に{"search_default_groupby_warehouse_id": 1}を追加する。だけ

できていない
- 元の検索ビューからinheritする必要があると思われる

### CODE REVIEW

**Comment by tack-yabeQuartile** [comment]
For now, this module is not recognized as a part of Stock app, so a new app is displayed after installing.

**Comment by yostashiro** [comment]
@kakurai8 @tack-yabeQuartile Should have advised earlier, but can you update this module https://github.com/qrtl/rpl-private/tree/12.0/stock_view_adj_rpl instead of adding a new one, since this change doesn't carry such a strong business context. It's a bit of an overkill to create a dedicated module.

File: stock_picking_type_kanban_view/views/stock_picking_type_kanban_view.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <field name="type">ir.actions.act_window</field>
+        <field name="inherit_id" ref="stock.stock_picking_type_action" />
+        <!-- <field name="view_type">list</field> -->
+        <field name="view_mode">kanban</field>
+        <field name="context">{"search_default_groupby_warehouse_id": 1}</field>
+    </record>
+    <menuitem
+        action="stock_picking_type_kanban_action"
+        id="stock_picking_type_kanban_action"
+    />
```
Comment (yostashiro) [inline stock_picking_type_kanban_view/views/stock_picking_type_kanban_view.xml:15]:
```suggestion
```
This shouldn't be needed.

File: stock_picking_type_kanban_view/views/stock_picking_type_kanban_view.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<odoo>
+    <record id="stock_picking_type_kanban_action" model="ir.actions.act_window">
+        <field name="name">Stock Picking Type Kanban Action</field>
+        <field name="res_model">stock.picking.type</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="inherit_id" ref="stock.stock_picking_type_action" />
+        <!-- <field name="view_type">list</field> -->
+        <field name="view_mode">kanban</field>
+        <field name="context">{"search_default_groupby_warehouse_id": 1}</field>
+    </record>
```
Comment (yostashiro) [inline stock_picking_type_kanban_view/views/stock_picking_type_kanban_view.xml:11]:
```suggestion
    <record id="stock.stock_picking_type_action" model="ir.actions.act_window">
        <field name="context">{"search_default_groupby_warehouse_id": 1}</field>
    </record>
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by tack-yabeQuartile** [comment]
> can you update stock_view_adj_rpl instead of adding a new one

OK, I checked the code after change and I feel the change is too small  to create a new custom module.

**Comment by tack-yabeQuartile** [comment]
moved to https://github.com/qrtl/rpl-private/pull/220

**Pull request closed without merging**